src/components/Modal/index.js

```diff
@@ -68,6 +71,20 @@ export function Modal({
                   style={closeButtonStyle}
                   className={`pointer-events-none sm:pointer-events-auto absolute top-0 right-0 ${closeButtonClass}`}
                 >
+                  {dropDownVisible && (
+                    <DropdownMenu
+                      className=""
+                      menuButtonClassName="p-0.5 min-w-7 rounded-full text-gray-400"
+                      renderButton={() => (
+                        <DotsVerticalIcon className="h-6 w-6 hover:text-gray-500 focus:outline-none" />
+                      )}
+                      items={[
```

Comment:
@OleksandrSekker how is another developer supposed to make these items links or at least be clickable?

What if you remake it to accept items as react node or node array?
```
const DropdownMenuItem = (props) => (<Menu.Item {...props}>{props.children({defaultClassName: "flex items-center w-full text-gray-700 hover:bg-gray-200"})}</Menu.Item>):
DropdownMenu.DropdownMenuItem = DropdownMenuItem;
```

And then, it seems to me, there will be enough control, but nevertheless, we do not duplicate some parts of the code, for example, those that are responsible for animation. @moshloop would you like such approach?
```
<DropdownMenu
          className=""
          menuButtonClassName="p-0.5 min-w-7 rounded-full text-gray-400"
          renderButton={() => (
            <DotsVerticalIcon className="h-6 w-6 hover:text-gray-500 focus:outline-none" />
          )}
          items={[
            <DropdownMenu.Item key="Duplicate">
              {({ defaultClassName }) =>
                (renderProps) =>
                  (
                    <button
                      type="button"
                      className={defaultClassName}
                      onClick={() => {}}
                    >
                      Duplicate
                    </button>
                  )}
            </DropdownMenu.Item>,
            <DropdownMenu.Item key="Share">
              <button
                type="button"
                className="className"
                onClick={() => {}}
              >
                Share
              </button>
            </DropdownMenu.Item>
          ]}
        />
```

Reply:
There is a TopologyDropdow that has a menu specific to topologies, Dropdown is a generic menu handler that accepts  menu items, we could add an option on it e.g. hamburgerStyle = "dots" that would render the 3 dots - However for this issue we don't have any menu items to display (the figma design is wrong, it should just be a close icon)

Reply:
this will be rethought the next time a dropdown with actions is needed, or a ready-made solution will be used if any.